content/history/contribute.inc

```diff
@@ -1,12 +0,0 @@
-<h1>Contribute</h1>
-
-<p>Please use one of the forms below to submit addition or corrections to the data we've got. It's just fine to submit multiple related pieces of information at once (e.g. the build dates of all your org's buggies). If you have a large quantity of data, or if you find it easier, you can also e-mail information to <a href="mailto:data@cmubuggy.org">data@cmubuggy.org</a>.</p>
```

Comment:
Just want to check whether Jeremy is cool with killing this?

Reply:
Yes.

![image](https://user-images.githubusercontent.com/7085856/149326639-7ec2ef1f-4b85-4193-a6e2-363dff3b5bfb.png)
![image](https://user-images.githubusercontent.com/7085856/149326678-67e8f150-f5f5-485f-9c86-7bfff332cd4a.png)